src/utils.js

```diff
@@ -38,6 +41,17 @@ const setupCml = async opts => {
       version !== 'latest' ? `@${version}` : ''
     }`
   );
+
+  let installedVersion = '';
+  try {
+    const json = await exec('npm list --json --global @dvcorg/cml');
+    installedVersion = JSON.parse(json).dependencies['@dvcorg/cml'].version;
+    console.log(
+      `Targeted CML version: ${version}, received: ${installedVersion}`
+    );
+  } catch (err) {
+    console.log('Failed to parse the install version of CML.', err);
```

Comment:
@0x2b3bfa0 would this err be worth failing the action for?

Reply:
🤷🏼‍♂️  Not being able to determine the installed version is a bit worrying, but not sure if there is any case where we should be tolerating it.

Reply:
Fail the setup action with a process.exit then?